src/assets/svg-components/MassaToken.tsx

```diff
@@ -0,0 +1,46 @@
+export interface MAssaTokenProps {
+  theme?: string;
```

Comment:
Why do we need to passa a theme!? 🤔

Reply:
I have the feeling that is not a good way!

I mean, not what you did but the way is being done. I think we should also have the images with a transparency or kind of and in that way it will took the background as color. Something in that sense...

We need to double check that before to go further!

Reply:
In my most recent commit I proposed the fix we talked about : 

Circle is div that follows tailwind presets, token svg has static color